src/ert/_c_wrappers/enkf/time_map.py

```diff
@@ -57,11 +57,15 @@ def fload(self, filename):
         if os.path.isfile(filename):
             OK = self._fload(filename)
             if not OK:
-                raise Exception(f"Error occured when loading timemap from:{filename}")
+                raise ValueError(f"Error occured when loading timemap from:{filename}")
         else:
             raise IOError((errno.ENOENT, f"File not found: {filename}"))
 
     def __eq__(self, other):
+        if (self is None and other is not None) or (self is not None and other is None):
```

Comment:
guess it works as intended - but it was not very nice :sweat_smile:

Reply:
how about
```
if any(self is None, other is None):                                          
    if all(self is None, other is None):
        return True                                                                                                                                                                                                    
    return False
```
instead of the existing lines 65-68? i agree the original description is meh

Reply:
or alternatively (no lint complaints):
```
if all(self is None, other is None):
    return True
if any(self is None, other is None):
    return False
```

Reply:
last solution implemented.